<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemWindow</class>
 <widget class="QMainWindow" name="SystemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>991</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QRadioButton" name="isTeam">
         <property name="text">
          <string>チーム戦</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>チーム1</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>プレイヤー人数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="team1_playerCount">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_40">
             <property name="text">
              <string>得点</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="team1_point">
             <property name="minimum">
              <number>-999</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>チーム名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="team1_name">
             <property name="text">
              <string>Aチーム</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="team1_area"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>チーム2</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>プレイヤー人数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="team2_playerCount">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_41">
             <property name="text">
              <string>得点</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="team2_point">
             <property name="minimum">
              <number>-999</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>チーム名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="team2_name">
             <property name="text">
              <string>Bチーム</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="team2_area"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="isIndividual">
         <property name="text">
          <string>個人戦</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>プレイヤー人数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="playerCount">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_4"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>得点リセット</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode1">
         <property name="text">
          <string>チーム点数表示（チーム戦）</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode2">
         <property name="text">
          <string>個人点数表示（チーム戦）</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode3">
         <property name="text">
          <string>個人点数表示（個人戦）</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode4">
         <property name="text">
          <string>虫食い画像</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode5">
         <property name="text">
          <string>逆書き漢字</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode6">
         <property name="text">
          <string>ファイブボンバー</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mode7">
         <property name="text">
          <string>漢字多答</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>isTeam</tabstop>
  <tabstop>team1_playerCount</tabstop>
  <tabstop>team1_point</tabstop>
  <tabstop>team1_name</tabstop>
  <tabstop>team2_playerCount</tabstop>
  <tabstop>team2_point</tabstop>
  <tabstop>team2_name</tabstop>
  <tabstop>isIndividual</tabstop>
  <tabstop>playerCount</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>mode1</tabstop>
  <tabstop>mode2</tabstop>
  <tabstop>mode3</tabstop>
  <tabstop>mode4</tabstop>
  <tabstop>mode5</tabstop>
  <tabstop>mode6</tabstop>
  <tabstop>mode7</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
